docs/installation/server-config/config-reference.rst

```diff
@@ -566,5 +568,8 @@ When deploying KSQL to production, the following settings are recommended in you
     # Bump the number of replicas for state storage for stateful operations
     # like aggregations and joins. By having two replicas (one main and one
     # standby) recovery from node failures is quicker since the state doesn't
-    # have to be rebuilt from scratch.
+    # have to be rebuilt from scratch. This configuration is also essential for
+    # pull queries to be highly available during node failures
     ksql.streams.num.standby.replicas=1
+
+For your convenience, a sample file is provided at ``<path-to-ksql-repo>/config/ksql-production-server.properties``
```

Comment:
People may not have cloned the KSQl repo, should we just put the URL here? https://github.com/confluentinc/ksql/tree/master/config

Reply:
I considered that. But what if this config changes based on versions? The installation directory seems to be only binaries?